src/Microsoft.ML.CodeGenerator/Microsoft.ML.CodeGenerator.csproj

```diff
@@ -0,0 +1,111 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <AssemblyName>Microsoft.ML.CodeGenerator</AssemblyName>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis" Version="2.10.0" />
```

Comment:
These version numbers should come from the [https://github.com/dotnet/machinelearning/blob/master/build/Dependencies.props](https://github.com/dotnet/machinelearning/blob/master/build/Dependencies.props) file. That way they can be shared with the .nupkgproj file.